### ISSUE TITLE
Add ability to use different fields for sorting in Filter panel

### ISSUE THREAD

**Issue by reupen** [reupen/columns_ui#25]

**Comment by reupen** [comment]
From https://hydrogenaud.io/index.php/topic,28647.msg917119.html#msg917119:

> I'd like to ask if there is a way to use separate columns to display and sort in the Filter panel? I'm not looking to ignore leading A's and The's, I'm looking for something similar to iTunes's "Sort Artist" or "Sort Album" field. The reason is my collection contains a good amount of Japanese songs, and titles containing kanji (Chinese characters) aren't sorting nicely. Right now, titles that start with kanji appear after those that start with hiragana/katakana glyphs, presumably sorted by their UTF code. Admittedly, because of how the language works, without supplying the phonetic reading of the words, the computer has no way to determine how a kanji glyph will be pronounced, and therefore sorted.
> 
> I have already tried creating a custom "SORT_ALBUM" tag and tried filtering/sorting using that tag, but while this works for the "Sort incoming files by:" option, it wreaks havoc with the filter panel. The problem is that it displays the value of SORT_ALBUM instead, which makes titles harder to read.
> 
> If it's not implemented yet, I hope this feature makes it into a future version of CUI!
> 
> tl;dr In the Filter panel, I'd like to display the value of one column (tag) but sort using another.

**Comment by reupen** [comment]
From https://github.com/reupen/columns_ui/issues/796#issuecomment-1702140467:

> But yes, ignoring articles, as well as sorting by romanized artist names while displaying normally. I would also like to sort my filter for albums by DATE/FULLDATE while having it simply display the year (or just the album name).
